ga4gh/cli.py

```diff
@@ -816,3 +820,72 @@ def client_main(parser=None):
     else:
         runner = args.runner(args)
         runner.run()
+
+##############################################################################
+# Configuration testing
+##############################################################################
+
+
+class TestConfigExists(unittest.TestCase):
+    """This class implements configuration checks for the GA4GH server. It is
+    structured as a set of unit tests, and relies on the unittest module.
+    However, as some of the tests require state injection (through command
+    line arguments) we need to break the usual unittest pattern and instead
+    implement our own simple test runner.
+    """
+
+    def __init__(self, parser=None):
+        self.parser = parser
+
+    def setUp(self):
+        self.app = flask.Flask(__name__)
+        parser = self.parser if self.parser is not None \
```

Comment:
The pep8 tool doesn't _currently_ have any issues with backslash line continuations, but pep8 is actually pretty clear on this:

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line  continuation.

So, I think we should avoid backslashes where we can. I think this is such an occasion. What about
```python
    parser = (
        self.parser 
        if self.parser is not None 
        else argparse.ArgumentParser(
            description="GA4GH server configuration validator"))
```
I quite like ternary operators, but I find it hard to write PEP8 compliant code using them, for just this reason. I would tend to favour the simpler
```python
    parser = self.parser
    if parser is None:
          parser = ArgumentParser(...)
```
in this situation.